components/service/experiments/README.md

```diff
@@ -155,67 +62,8 @@ Experiments allows you to specify the following filters:
 - lang (regex): language, pulled from the default locale
 - device (regex): Android device name
 - manufacturer (regex): Android device manufacturer
-- region: custom region, different from the one from the default locale (like a GeoIP, or something similar).
-- release channel: release channel of the app (alpha, beta, etc)
-
-For region and release channel to work you must provide a `ValuesProvider` implementation when creating the `Experiments` instance, as detailed below
-
-### Specifying custom values for filters
-Additionally, Experiments allows you to specify a custom `ValuesProvider` object in order to return a custom region,
-different from the one of the current locale (perhaps doing a GeoIP or something like that), or the app
-relase channel (alpha, beta, etc). It also allows you to override the values for other experiment properties
-(such as the appId, country, etc):
-
-```Kotlin
-val experiments = Experiments(
-    experimentSource,
-    experimentStorage,
-    object : ValuesProvider {
-        override fun getRegion() {
-            return custom_region
-        }
-
-        override fun getReleaseChannel() {
-            return app_channel
-        }
-    }
-)
-```
-
-### Creating a custom experiment source
-You can create a custom experiment source simply by implementing the `ExperimentSource` interface:
-
-```Kotlin
-class MyExperimentSource : ExperimentSource {
-    override fun getExperiments(snapshot: ExperimentsSnapshot): ExperimentsSnapshot {
-        // ...
-        return updatedSnapshot
-    }
-}
-```
-
-The `getExperiments` method takes an `ExperimentsSnapshot` object, which contains the list of already downloaded experiments and 
-a last_modified date, and returns another `ExperimentsSnapshot` object with the updated list of experiments.
-
-As the `getExperiments` receives the list of experiments from storage and a last_modified date, it allows you
-to do diff requests, if your storage mechanism supports it (like Kinto does).
-
-### Creating a custom experiment storage
-You can create a custom experiment storage simply by implementing the `ExperimentStorage` interface, overriding
-the save and retrieve methods, which use `ExperimentsSnapshot` objects with the list of experiments and a last_modified date:
-
-```Kotlin
-class MyExperimentStorage : ExperimentStorage {
-    override fun save(snapshot: ExperimentsSnapshot) {
-        // save snapshot to disk
-    }
-
-    override fun retrieve(): ExperimentsSnapshot {
-        // load snapshot from disk
-        return snapshot
-    }
-}
-```
+- *(TBD: region: custom region, different from the one from the default locale (like a GeoIP, or something similar).)*
```

Comment:
nit: is there any bug filed for these TBD that you can link here?

Reply:
Not at this point, it's an open question/conversation in the planning doc.